app/scripts/HiGlassComponent.js

```diff
@@ -188,8 +188,16 @@ class HiGlassComponent extends React.Component {
       setTileProxyAuthHeader(props.options.authToken);
     }
 
+    this.pixiRoot = new PIXI.Container();
+    this.pixiRoot.interactive = true;
```

Comment:
Does pixiStage also need to be interactive if `pixiRoot` is interactive?

Come to think of it, why do either of them have to be interactive? Do we use PIXI events anywhere?

Reply:
I believe both have to be interactive. Maybe the root container does not. I didn't notice any performance impact but shall I test if `this.pixiRoot` can also be non-interactive?

We use PIXI events for certain annotation tracks which support click event callbacks.